src/NumericalAlgorithms/FiniteDifference/HighOrderFluxCorrection.hpp

```diff
@@ -163,9 +188,17 @@ void cartesian_high_order_fluxes_using_nodes(
          ++storage_index) {
       const auto flux_multi_index = prepend(
           second_order_var_correction.get_tensor_index(storage_index), dim);
+      std::array<size_t, Dim> flux_storage_indices;
+      for (size_t n = 0; n < Dim; ++n) {
+        auto temp_multi_index = prepend(
+            second_order_var_correction.get_tensor_index(storage_index), n);
+        flux_storage_indices[n] =
+            FluxTensor::get_storage_index(temp_multi_index);
+      }
       const size_t flux_storage_index =
           FluxTensor::get_storage_index(flux_multi_index);
       // Loop over each face
+      auto current_normal = effective_normal[dim];
```

Comment:
Did you really mean to copy data here? I would've guessed this should be a `const auto&`